pylint/plugins/hass_imports.py

```diff
@@ -131,6 +131,10 @@ class ObsoleteImportMatch:
             reason="replaced by ColorMode enum",
             constant=re.compile(r"^COLOR_MODE_(\w*)$"),
         ),
+        ObsoleteImportMatch(
+            reason="replaced by color modes",
+            constant=re.compile("^SUPPORT_(BRIGHTNESS|COLOR_TEMP|COLOR|WHITE_VALUE)$"),
```

Comment:
This can be simplified once https://github.com/home-assistant/core/pull/76926 is merged
```suggestion
            constant=re.compile("^SUPPORT_(BRIGHTNESS|COLOR_TEMP|COLOR)$"),
```

Reply:
this is now ready :)